resources/views/resource/index.blade.php

```diff
@@ -20,7 +20,37 @@
             <p class="flex">
                 <a href="{{ route('resource.create') }}" class="btn">Create New</a>
                 <a href="{{ route('resource.json') }}" class="btn">Create from JSON</a>
+                <!--UI to show the Delete button-->
```

Comment:
None of the changes to this file are appropriate for this PR.

Reply:
PR removed. I apologize for it.